templates/country_selector_banner.twig

```diff
@@ -0,0 +1,13 @@
+<div id="country-selector-banner" class="collapse">
```

Comment:
Similar here. We would be adding this markup to every page of every site, and it will only be used on a single page of a single site.